packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -310,25 +330,50 @@ export class Stepper implements LocalizedComponent, T9nComponent {
     updateMessages(this, this.effectiveLocale);
   }
 
-  private itemMap = new Map<HTMLCalciteStepperItemElement, { position: number; content: Node[] }>();
+  /** keep track of the currently active item position */
+  @State() currentPosition: number;
 
-  /** list of sorted Stepper items */
-  private items: HTMLCalciteStepperItemElement[] = [];
+  @Watch("currentPosition")
+  handlePositionChange(): void {
+    requestAnimationFrame(() => {
+      this.determineActiveStepper(true);
+    });
+  }
+
+  @State() elWidth: number;
+
+  @Watch("elWidth")
+  handleElWidthChange(): void {
+    requestAnimationFrame(() => {
+      this.determineActiveStepper();
+    });
+  }
 
   /** list of enabled Stepper items */
```

Comment:
[Unrelated] I think we can drop this comment as the variable name is clear.

Reply:
Dropped.